watchtower/wtdb/client_db.go

```diff
@@ -951,6 +961,33 @@ func (c *ClientDB) RegisterChannel(chanID lnwire.ChannelID,
 			return err
 		}
 
+		// Get the channel-id-index bucket.
+		indexBkt := tx.ReadWriteBucket(cChanIDIndexBkt)
+		if indexBkt == nil {
+			return ErrUninitializedDB
+		}
+
+		// Request the next unique id from the bucket.
+		nextSeq, err := indexBkt.NextSequence()
```

Comment:
W/ the comment above, we can ignore the sequencing here and just use the scid itself. The tradeoff here is that it'll propagate a larger API change across all relevant callers.